index.html

```diff
@@ -56,238 +57,248 @@ <h3>LET'S CONNECT</h3>
       </footer>
     </div>
     </div>
-  </section>
-
-  <section id="work_outline">
-
-    <!-- Card1 -->
-    <div class="row-example">
-      <div class="column-example ">
-        <div class="all_project-images proj-image1"></div>
-      </div>
-      <div class="column-example">
-        <h1>Multi-Post
-          Stories</h1>
-        <div id="general_inf_3">
-          <p class="title">CANOPY</p>
-          <img class="counter" src="./images/Counter (1).svg" alt="counter" />
-          <p class="job-title">Back End Dev</p>
-          <img class="counter" src="./images/Counter (1).svg" alt="counter">
-          <p class="year">2015</p>
-
-        </div>
-        <p class="project_description">
-          A daily selection of privately personalized reads; no accounts or sign-ups required.
-        </p>
-
-        <footer>
-          <ul id="skills_3" class="skills">
-            <li>html</li>
-            <li>css</li>
-            <li>javaScript</li>
-          </ul>
-          <div class="external_link">
-            <a href="#">See Project</a>
-
-          </div>
-        </footer>
+    <nav class="container">
+      
+      <div class="menu-wrapper">
+        <img id="mobile-menu" class="mobile-menu" src="./images/Union.svg" alt="open navigation">
       </div>
+      <div id="nav-list-menu" class="nav-list-menu">
+        <i class="fa fa-fw fa-x mobile-menu-exit" id="mobile-menu-exit"></i>
+        <ul id="nav-list" class="nav-list">
+        <li class="nav-item"><a href="#portfolio">Portfolio</a></li>
+        <li class="nav-item"><a href="#about">About</a></li>
+        <li class="nav-item"><a href="#contact">Contact</a></li>
+      </ul>
     </div>
```

Comment:
- [ ] To meet the design requirements of this milestone, you are required to have a "blurred" design of your background when the background of the mobile menu is opened. For some context of how you can achieve this, kindly have a look at [this](https://blog.prototypr.io/how-to-make-a-background-blur-in-css-with-one-line-of-code-e446c7236e60) resource.
- [ ] Also, kindly increase the margins/ padding top and bottom of your individual menu links so that it matches the provided Figma design.

**This is how your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/203745167-7ab3d4b8-063b-4c3b-957b-d9d0cacbb236.png)

**This is the desired design from the Figma design:**

![image](https://user-images.githubusercontent.com/86318284/203745299-2d9da96d-91c6-4765-a0af-b17e71540210.png)